Gazebo new vehicles, ROS 2 frame conventions, multi vehicle simulation (#2396)
PX4/PX4-user_guide: en/sim_gazebo_gz/multi_vehicle_simulation.md

  - When used with `PX4_GZ_MODEL`, Gazebo will automatically pick a unique model name in the form `${PX4_GZ_MODEL}_instance`. 
- `ARGS`:
   A list of environmental variables, as described in [Gazebo Simulation > Usage/Configuration Options](../sim_gazebo_gz/README.md#usage-configuration-options).

This allows for greater flexibility and customization.

## Multiple Vehicles with ROS 2 and Gazebo

[Multiple vehicles with ROS 2](../ros/ros2_multi_vehicle.md) are possible.

- First follow the installation instructions for [Gazebo](../sim_gazebo_gz/README.md).
- Then configure your system for [ROS 2 / PX4 operations](../ros/ros2_comm.md#installation-setup).
- In different terminals manually start a multi vehicle simulation.

>   This example spawns 2 x500 quadrotors and a fixed wing

Reviewer: ```suggestion
  This example spawns 2 X500 Quadrotors and a fixed wing.
```

  ```sh
  PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4 -i 1
  ```
  ```sh
  PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_POSE="0,1" PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4 -i 2
  ```
  ```sh
  PX4_SYS_AUTOSTART=4004 PX4_GZ_MODEL_POSE="0,2" PX4_GZ_MODEL=rc_cessna ./build/px4_sitl_default/bin/px4 -i 3
  ```
- Start the agent
  ```sh
  MicroXRCEAgent udp4 -p 8888
  ```

> - The agent will automatically connect to all clients and running `ros2 topic list` will return

Reviewer: ```suggestion
  The agent will automatically connect to all clients.
- Run `ros2 topic list`. This should return:
```

  - When used with `PX4_GZ_MODEL`, Gazebo will automatically pick a unique model name in the form `${PX4_GZ_MODEL}_instance`. 
- `ARGS`:
   A list of environmental variables, as described in [Gazebo Simulation > Usage/Configuration Options](../sim_gazebo_gz/README.md#usage-configuration-options).

This allows for greater flexibility and customization.

## Multiple Vehicles with ROS 2 and Gazebo

[Multiple vehicles with ROS 2](../ros/ros2_multi_vehicle.md) are possible.

- First follow the installation instructions for [Gazebo](../sim_gazebo_gz/README.md).
- Then configure your system for [ROS 2 / PX4 operations](../ros/ros2_comm.md#installation-setup).
- In different terminals manually start a multi vehicle simulation.
  This example spawns 2 X500 Quadrotors and a fixed wing:

  ```sh
  PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4 -i 1
  ```
  
  ```sh
  PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_POSE="0,1" PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4 -i 2
  ```
  
  ```sh
  PX4_SYS_AUTOSTART=4004 PX4_GZ_MODEL_POSE="0,2" PX4_GZ_MODEL=rc_cessna ./build/px4_sitl_default/bin/px4 -i 3
  ```
  
- Start the agent:

  ```sh
  MicroXRCEAgent udp4 -p 8888
  ```

> - The agent will automatically connect to all clients and running `ros2 topic list` will return:
> - 

Reviewer: ```suggestion
  The agent will automatically connect to all clients.
-  Run `ros2 topic list` to see the topic list, which should look like this:
```
